cleaner.py

```diff
@@ -36,10 +37,31 @@ def parse_time(s: str):
                 print('** DRY RUN **')
             else:
                 job.delete()
+            continue
+    start_time = parse_time(job.obj['status'].get('startTime'))
+    seconds_since_start = now - start_time
+    annotations = job.obj['metadata'].get('annotations')
+    if annotations is None:
+        continue
```

Comment:
I guess the `--timeout-seconds` flag should still work even if no annotation is specified?